index.html

```diff
@@ -30,7 +31,17 @@
             </li>
           </ul>
         </nav>
-
+        <ul class="desk-menu">
+          <li>
+              <a href="#Portfolio" class="navnav-item">Portfolio</a>
+          </li>
+          <li>
+              <a href="#About">About</a>
+          </li>
+          <li>
+              <a href="#Contact">Contact</a>
+          </li>
+      </ul>
```

Comment:
- [ ] KIndly make sure when one of the navigation links is clicked we have redirected to the part page that the button represents. If we click on the `About` link, the page goes to the `About-myself` section, and make sure the menu disappears immediately.

- [ ] Kindly make sure when the menu is opened, even if we scroll down we will continue to see all navigation links until we close the menu, at this moment when we scroll down while the menu is opened the `Close` menu button disappears. Kindly fix that to match all requirements.

  - ## APP BEHAVIOUR
  
   ![chrome-capture-2023-2-7 (2)](https://user-images.githubusercontent.com/87197412/223355178-91290b15-3264-4d8a-8cd8-d3b8983db86f.gif)